manifest.json

```diff
@@ -113,6 +113,7 @@
                 "plugins/sendvid_a.js",
                 "plugins/squarespace_a.js",
                 "plugins/streamable_a.js",
+                "plugins/tapas.js",
```

Comment:
You don't need to load this plugin for all sites, only for tapas.io.

Reply:
Good eyes, my oversight. Squared away.